includes/discovery/ports/zynos.inc.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/*
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ * @author     LukeKelsall <l.kelsall@b4rn.org.uk>
+ */
+
+//loop through ports
+foreach ($port_stats as $index => $port) {
+    //Logically convert swp ports to human readable
+    //only convert swp ports
+    if (substr($port['ifName'], 0, 3) === 'swp') {
+        $portNum = preg_replace('/[a-z]+/', '', $port['ifName']);
+        if ($portNum < 50) {
+            $portNum++;
+            //Leading 0 for single digits
+            if ($portNum < 10) {
```

Comment:
instead of doing that, you can just do this:
```php
$ifDescr = '1/' . zeropad($portNum);
```

Reply:
Never used zeropad but it looks interesting, I will test it and get back to you.

Reply:
Zeropad works really well, Thanks for pointing this out :)